packages/frontmatter/src/downloads.tsx

```diff
@@ -1,6 +1,5 @@
 import { Menu } from '@headlessui/react';
-import DocumentIcon from '@heroicons/react/24/outline/DocumentIcon';
-import ArrowDownTrayIcon from '@heroicons/react/24/outline/ArrowDownTrayIcon';
+import { DocumentIcon, ArrowDownTrayIcon } from '@heroicons/react/24/outline';
```

Comment:
This is the one potential problem, we may be bringing in all icons, however, I think that ESM should fix this.

Reply:
I can confirm that this is fixed with ESM:
![image](https://github.com/executablebooks/myst-theme/assets/913249/2470fb26-c955-41d3-9dbd-61ac1c80ce8f)